core/src/Repository/DataSource/Sql/Queries/UserSqlSchema.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Harmony\Core\Repository\DataSource\Sql\Queries;
+
+use Harmony\Core\Repository\DataSource\Sql\Helper\DefaultSqlSchema;
+
+class UserSqlSchema extends DefaultSqlSchema {
+  public const COLUMN_NAME = "landing_page_id";
+  public const COLUMN_EMAIL = "use_facebook_ads";
```

Comment:
This is project specific

Reply:
@lucianosantana I didn't push a change that I hadn't staged. Bud the thing about UserSqlSchema and the two queries I created is to show the usage, they will be removed, please notice that this PR is in draft because of this. It's purpose is to discuss it during tomorrow's meeting.

Reply:
I'm resolving the conversation since the comment does not apply

Reply:
👍 Understood. I was actually wondering what would be the appropriate place to have those examples. Because it seems to more like a hypothetical sample than something intended to be used. So I'd have it integrated with the example we have on the sample folder.